server/protocols/jmap/src/main/java/org/apache/james/jmap/model/Message.java

```diff
@@ -130,12 +132,13 @@ private static String getPreview(IndexableMessage im) {
     }
 
     @VisibleForTesting static String computePreview(String body) {
-        if (body.length() <= 256) {
-            return body;
+        String bodyWithoutTag = Jsoup.parse(body).text();
```

Comment:
it's just a strait SRP application, there's no much debate to have about that. Generating a preview should not be a Message responsibility.